Form/Type/SearchType.php

```diff
@@ -17,7 +19,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'term',
-                'text',
+                TextType::class,
```

Comment:
ik vind da wel nuttig, die minimum php versie in je composer.json, al is het maar om op packagist/github snel te zien wat de minimum vereiste is.